packages/react-components/react-card/package.json

```diff
@@ -53,8 +53,7 @@
   "beachball": {
     "disallowedChangeTypes": [
       "major",
-      "minor",
-      "patch"
+      "prerelease"
```

Comment:
just check to make sure that there are no prerelease changefiles for react-card, otherwise this will break release :)